api.py

```diff
@@ -115,6 +115,10 @@ def dispatcher(router, request, response):
         response.write(json.dumps(rv, default=bson.json_util.default))
         response.headers['Content-Type'] = 'application/json; charset=utf-8'
 
+try:
+    from newrelic import agent
```

Comment:
The ```import newrelic.agent``` style is generally preferable over the ```from newrelic import agent``` style for code readability and traceability reasons.